src/backend/joanie/tests/core/test_api_enrollment.py

```diff
@@ -202,6 +202,85 @@ def test_api_enrollment_read_list_authenticated(self):
             },
         )
 
+    def test_api_enrollment_read_list_filtered_by_course_run_id(self):
```

Comment:
Just curious: can we filter with several course run ids? Or will you send as many requests as you have products?

Reply:
Nope currently this filter does not support a list of course run id.
About products, enrollments are nested within order.

Reply:
:+1: